internal/engine/generate/sql.go

```diff
@@ -10,11 +10,16 @@ import (
 // It can optionally rewrite named parameters to numbered parameters.
 // If so, it returns the order of the parameters in the order they appear in the statement.
 func WriteSQL(node *parse.SQLStatement, orderParams bool, pgSchema string) (stmt string, params []string, err error) {
+	if node == nil {
+		panic("statement ast node is nil")
```

Comment:
There's a recover just below that converts to an error return.  Why hard panic here?

Reply:
I don't remember why I did this. Will change